packages/frontend/src/components/table/ProjectNameCell.tsx

```diff
@@ -15,19 +15,28 @@ export interface ProjectCellProps {
     isUpcoming?: boolean
     isArchived?: boolean
     redWarning?: string
+    type?: 'layer2' | 'layer3' | 'bridge' | undefined
     showProjectUnderReview?: boolean
     hasImplementationChanged?: boolean
     warning?: string
     data?: { syncStatus?: SyncStatus }
   }
+  showIsL3?: boolean
```

Comment:
Why do we need this `showIsL3` when we do `showIsL3 && project.type === 'layer3'` anyway?